KotlinFutures/app/src/main/java/com/example/kotlinfutures/viewmodel/DetailViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.kotlinfutures.viewmodel
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import com.example.kotlinfutures.model.Repository
+import com.example.kotlinfutures.model.RepositoryDefault
+import com.example.kotlinfutures.model.User
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+
+class DetailViewModel : ViewModel() {
+
+    val repository: Repository = RepositoryDefault()
+
+    lateinit var userData: MutableLiveData<User>
+    fun getData(): MutableLiveData<User> {
```

Comment:
It should return only the instance of userData, there should not be some logic for loading data